core/src/test/java/org/elasticsearch/discovery/DiscoveryWithServiceDisruptionsTests.java

```diff
@@ -240,12 +240,16 @@ public void failWithMinimumMasterNodesConfigured() throws Exception {
     /** Verify that nodes fault detection works after master (re) election */
     @Test
     public void testNodesFDAfterMasterReelection() throws Exception {
-        startCluster(3);
+        startCluster(4);
```

Comment:
In the end of the tests we brought the cluster down to one node, which required setting min master node to 1 (which we weren't doing). I felt uncomfortable setting it to 1, I chose to start with 4 nodes bringing the cluster down to 2 (and min master node to 2 as well).